lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -244,7 +254,8 @@ const cli = yargs
   .help('help')
   .describe({
     'config-path': 'The path to the config JSON file',
-    'expectations-path': 'The path to the expected audit results file'
+    'expectations-path': 'The path to the expected audit results file',
+    'save-assets-to': 'The path to save the assets to',
```

Comment:
`'save-assets-path'`? description could be "Saves run assets to this path" or something to be clear that it's not saved usually